apps/api/src/app/inbox/usecases/mark-many-notifications-as/mark-many-notifications-as.usecase.ts

```diff
@@ -0,0 +1,66 @@
+import { Injectable } from '@nestjs/common';
+import {
+  buildFeedKey,
+  buildMessageCountKey,
+  InvalidateCacheService,
+  WebSocketsQueueService,
+} from '@novu/application-generic';
+import { MessageRepository } from '@novu/dal';
+import { WebSocketEventEnum } from '@novu/shared';
+
+import { ApiException } from '../../../shared/exceptions/api.exception';
+import { GetSubscriber } from '../../../subscribers/usecases/get-subscriber';
+import { MarkManyNotificationsAsCommand } from './mark-many-notifications-as.command';
+
+@Injectable()
+export class MarkManyNotificationsAs {
```

Comment:
This use case is created with the future in mind. It allows you to mark many notifications with status. If we decide to expose the API to mark many notifications with status, it will be easy to do.

It's used for marking a single notification as well.